### ISSUE TITLE
Make Intercon COVID Information more visible

### ISSUE THREAD

**Issue by diewald** [neinteractiveliterature/intercode#5313]
Right now, this is five sections down on the Convention Rules page. I had to search to find it, and I _knew_ it was there. This really needs to be more visible. 

I would suggest:

1. A new entry belongs on the top menu bar. Other events will probably want this as well.
2. A clear and blatant pointer definitely belongs on the home page. 
3. Given the uncertainty around COVID, I'd also suggest that the COVID information gets its own page. We know it's going to grow, because we want to add the link to today's Q&A - and we need to plan for change as the pandemic itself changes.

Just my thoughts as I was trying to point someone else to the information. 

Thanks!

**Comment by dkapell** [comment]
All of this is in the works, as Alison Joy mentioned on today's call.  I'll be working with her over the next month or so to make sure all this information is readily available.

**Comment by dkapell** [comment]
As far as other events, the built in Content Management System allows Event Admins to put whatever they want on the top level, so that functionality exists today.

**Comment by diewald** [comment]
@dkapell I've have no concerns about getting the content added. My concern is about the visibility of that content. So does "allows Event Admins to put whatever they want on the top level" means I should suggest this to Alison?

**Comment by dkapell** [comment]
That part was in response to the suggestion that other events would want a
top level section.  They can use the tools in the webapp to build that
themselves, same as any top level section other than events/admin.

 I have been assigned this task in my role as part of the web team for
Intercon U, so there's no need to also suggest this to Alison.  If there
are other events that are using intercode (say a bubble) that you want to
suggest it to, feel free to do so.

**Comment by nbudin** [comment]
I was chatting with Alison Joy about this earlier and she gave me some pointers on what she wanted to see for the site, so I went ahead and made the changes.  There already was a COVID FAQ page that she'd put together, and I did a couple things to make it more visible.  Here's what's new:

1. There's a top-level navigation menu item linking directly to the FAQ page.
2. There's also a news item under "convention news" on the front page announcing that the FAQ exists, and linking to it.
